code/game/turfs/closed/_closed.dm

```diff
@@ -69,12 +69,28 @@ INITIALIZE_IMMEDIATE(/turf/closed/indestructible/splashscreen)
 	if(SStitle.icon)
 		icon = SStitle.icon
 
+	update_splash()
+
+
+/turf/closed/indestructible/splashscreen/proc/update_splash()
+	var/splash_text = pick(strings(SPLASH_DESC_FILE, "splashes"))
+	maptext = MAPTEXT("<font color='ebfc50'>[splash_text]</font>")
+	maptext_height = world.icon_size * 4
+	maptext_width = world.icon_size * 16
+	var/list/offsets = get_title_offset()
+	maptext_x = offsets[1]
+	maptext_y = offsets[2]
+
```

Comment:
rotate the maptext here 45 degrees
use animate() to make it get slightly bigger/smaller like minecraft

Reply:
how do i rotate and how do i animate it

Reply:
Rotate is a matrix proc and animate is just animate(). Check the byond documentation for more details.
If you have the DM language client installed in VS code, you can control click on byond functions to view their documentation, or you can find the docs online. They are basic, but informative.

Reply:
well, would that work with maptext is the problem,

Reply:
You would have to rotate whatever the maptext is attached to, which would require a holder effect object for the text

Reply:
holder effect object?

Reply:
can it just be flat?